Main.java

```diff
@@ -0,0 +1,78 @@
+import java.util.*;
+import java.io.*;
+
+class Main{
+    public static void main(String []args) throws IOException{
+        int endPosition = 100;
+        boolean isFinished = false;
+        File file=new File("Input.txt");    //creates a new file instance  
+        FileReader fr=new FileReader(file);
+        BufferedReader br = new BufferedReader(fr);//(new InputStreamReader(System.in));
```

Comment:
There is no entity modelling in this solution. Please think in terms of OO Design such as having classes like
Board, Player, Ladder, Snake, Dice, Game. Try defining the responsibility of each and every entity and their corresponding actions.